telegram/inline/inlinekeyboardbutton.py

```diff
@@ -23,35 +23,49 @@
+        callback_game (:class:`telegram.CallbackGame`): Optional. Description of the game that will
+            be launched when the user presses the button.
+        pay (:obj:`bool`): Optional. Specify True, to send a Pay button.
 
     Args:
-        text (str): Label text on the button.
-        url (Optional[str]): HTTP url to be opened when button is pressed.
-        callback_data (Optional[str]):  Data to be sent in a callback query to the bot when button
-            is pressed, 1-64 bytes.
-        switch_inline_query (Optional[str]): If set, pressing the button will prompt the user to
-            select one of their chats, open that chat and insert the bot's username and the
+        text (:obj:`str`): Label text on the button.
+        url (:obj:`str`): HTTP url to be opened when button is pressed.
+        callback_data (:obj:`str`, optional): Data to be sent in a callback query to the bot when
+            button is pressed, 1-64 bytes.
+        switch_inline_query (:obj:`str`, optional): If set, pressing the button will prompt the
+            user to select one of their chats, open that chat and insert the bot's username and the
             specified inline query in the input field. Can be empty, in which case just the bot's
-            username will be inserted.
-        switch_inline_query_current_chat (Optional[str]): If set, pressing the button will insert
-            the bot's username and the specified inline query in the current chat's input field.
-            Can be empty, in which case only the bot's username will be inserted.
-        callback_game (Optional[:class:`telegram.CallbackGame`]): Description of the game that will
-            be launched when the user presses the button. NOTE: This type of button must always be
-            the first button in the first row.
-        pay (Optional[bool]): Specify True, to send a Pay button. NOTE: This type of button must
-            always be the first button in the first row.
-        **kwargs (dict): Arbitrary keyword arguments.
-
+            username will be inserted. This offers an easy way for users to start using your bot
+            in inline mode when they are currently in a private chat with it. Especially useful
+            when combined with switch_pm* actions - in this case the user will be automatically
+            returned to the chat they switched from, skipping the chat selection screen.
+        switch_inline_query_current_chat (:obj:`str`, optional): If set, pressing the button will
+            insert the bot's username and the specified inline query in the current chat's input
+            field. Can be empty, in which case only the bot's username will be inserted. This
+            offers a quick way for the user to open your bot in inline mode in the same chat - good
+            for selecting something from multiple options.
+        callback_game (:class:`telegram.CallbackGame`, optional): Description of the game that will
```

Comment:
I guess they added it but did not imlpement it. I can add a note to the docstring.